app/bundles/LeadBundle/Controller/Api/LeadApiController.php

```diff
@@ -565,6 +565,18 @@ public function getEventsAction($id)
         return $this->handleView($this->view($events));
     }
 
+    /**
+     * Creates new entity from provided params.
+     *
+     * @param array $params
+     *
+     * @return object
+     */
+    public function getNewEntity(array $params)
```

Comment:
Why is the name `getNewEntity` if this method can return existing entity?
Is this method necessary to be in the Controller?